internal/tiltfile/files.go

```diff
@@ -10,11 +10,13 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/tilt-dev/tilt/internal/engine/local"
```

Comment:
our dependency graph is going to get weird if the Tiltfile execution controller depends on the Local Resource execution controller

Maybe we need a third package for these kinds of command APIs that both packages can depend on? or just put the new ExecLocalCmd code in pkg/model?

Reply:
Absolutely, I was conflicted about where to put this and meant to revisit. I think a new package is probably warranted - I'm hesitant to add to `pkg/model` since it's an internal construct for execution and shouldn't really be part of public API surface

Reply:
Moved to `internal/localexec` in cdeef139b0b444b7088f9da26caa6a7dac9257e7